File: rust/testdata/capabilities/Probe.java
package probe;

import com.example.Helper;

public class Widget extends BaseWidget implements Renderable {
    private String name;

    public String render() {
        return Helper.help(this.name);
    }

    public static void main(String[] args) {
        Widget w = new Widget();
        w.render();
    }
}

File: rust/testdata/capabilities/probe.php
<?php
namespace Probe;

use Example\Helper;

class Widget extends BaseWidget implements Renderable
{
    private $name;

    public function render()
    {
        return Helper::help($this->name);
    }
}

function main()
{
    $w = new Widget();
    $w->render();
}
